modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableMessageGroup.java

```diff
@@ -154,5 +155,8 @@ interface Commands {
 
         /** Message type for {@link TablePartitionIdMessage}. */
         short TABLE_PARTITION_ID = 61;
+
+        /** Message type for {@link BuildIndexCommand}. */
+        short BUILD_INDEX = 62;
```

Comment:
I guess you didn't notice the pattern. Here commands are 40 and higher. 60+ is for random stuff, like TablePartitionId pojos. Please make it 44 and move it higher.